source/common/http/http3/codec_stats.h

```diff
@@ -11,18 +11,13 @@ namespace Http3 {
 
 /**
  * All stats for the HTTP/3 codec. @see stats_macros.h
- * TODO(danzh) populate all of them in codec.
```

Comment:
Do we have to remove these? I planned to populate them gradually.

Reply:
I don't think leaving them in the code base while they're not being used makes sense, but I think it'd be fine to add them back (and doc them up) as they're implemented.